### PR TITLE
HKG: Car Port for Kia K5 Hybrid 2020

### PR DESCRIPTION
**Pull request by sunnyhaibin**
**Checklist**
- [x] added entry to CarInfo in selfdrive/car/*/values.py and ran `selfdrive/car/docs.py` to generate new docs
- [x] test route added to [routes.py](https://github.com/commaai/openpilot/blob/master/selfdrive/car/tests/routes.py)
- [x] route with openpilot: `c58dfc9fc16590e0|2023-01-14--13-51-48`
- [x] Harness Type: [Hyundai A](https://github.com/commaai/openpilot/wiki/Hyundai-Kia-Genesis#harness-guide)

Thanks to community 2020 Kia K5 Hybrid owner `NeoFaust#9519` (Discord).

### CODE REVIEW

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -1105,6 +1107,31 @@ class Buttons:
...
+    (Ecu.fwdRadar, 0x7D0, None): [
+      b'\xf1\x00DLhe SCC FHCUP      1.00 1.02 99110-L7000         ',
+    ],
+    (Ecu.eps, 0x7D4, None): [
+      b'\xf1\x00DL3 MDPS C 1.00 1.02 56310-L7000 4DLHC102',
+    ],
+    (Ecu.fwdCamera, 0x7C4, None): [
+      b'\xf1\x00DL3HMFC  AT KOR LHD 1.00 1.02 99210-L2000 200309',
+    ],
+    (Ecu.abs, 0x7D1, None): [
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:1120]: can you provide routes for these versions?

Comment (sunnyhaibin) [reply selfdrive/car/hyundai/values.py:1122]: Yep! `c58dfc9fc16590e0|2023-01-14--13-51-48`

Comment (sshane) [reply selfdrive/car/hyundai/values.py:1122]: Where did all the ABS and the other transmission come from?

Comment (sunnyhaibin) [reply selfdrive/car/hyundai/values.py]: Oops, I think I accidentally copied it from another platform. The transmission FW is from `c58dfc9fc16590e0|2023-01-14--00-01-18--2`

**Comment by sunnyhaibin** [comment]
Radar tracks are enabled via script and found on this car: `c58dfc9fc16590e0|2023-01-14--17-37-44`

**Comment by adeebshihadeh** [comment]
@sshane anything blocking this?

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -1148,6 +1150,24 @@ class Buttons:
...
+    ],
+    (Ecu.fwdCamera, 0x7C4, None): [
+      b'\xf1\x00DL3HMFC  AT KOR LHD 1.00 1.02 99210-L2000 200309',
+    ],
+    (Ecu.engine, 0x7E0, None): [
+      b'\xf1\x87391162JLA0',
+    ],
+    (Ecu.transmission, 0x7E1, None): [
+      b'\xf1\x00PSBG2323  E08\x00\x00\x00\x00\x00\x00\x00TDL2H20KA2\xe3\xc6cz',
+      b'\xf1\x87959102T250\x00\x00\x00\x00\x00\xf1\x81E08\x00\x00\x00\x00\x00\x00\x00\xf1\x00PSBG2323  E08\x00\x00\x00\x00\x00\x00\x00TDL2H20KA2\xe3\xc6cz',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:1168]:
```suggestion
```

**Pull request merged**